internals/maintenance_scripts.py

```diff
@@ -514,17 +514,12 @@ def handle_creation(self, stage: Stage, stage_dict: StageDict) -> str | None:
     """Send a flagged creation request for processing to the Origin Trials
     API.
     """
-    request_success = False
-    attempted_requests = 0
     new_id = None
-    while not request_success and attempted_requests < 3:
-      attempted_requests += 1
-      try:
-        new_id = origin_trials_client.create_origin_trial(stage)
-        request_success = True
-      except requests.RequestException:
-        attempted_requests += 1
-    if not request_success:
+    # TODO(DanielRyanSmith): This request should have a retry process based on
+    # the error code returned from the OT server.
+    try:
+      new_id = origin_trials_client.create_origin_trial(stage)
```

Comment:
Checking the return response of Trials on RPCstudio, I think new_id is of int type?

Reply:
That might be the case - it's returned as a string from the ListTrials endpoint. I can update this to anticipate an int data type

Reply:
I checked ListTrials endpoint and it also returns int. The string type has "" around it. Could you double check?

Reply:
I'm not certain, since a direct RPC returns an int value, but when fetched from the OT API through Chromestatus, it's a string.  Either way, it's safer in this case to just assume that the data type is an int and convert from there